src/ansys/aedt/core/filtersolutions_core/dll_interface.py

```diff
@@ -59,7 +59,7 @@ def _init_dll_path(self, version):
             version + "CL" in aedt_versions.installed_versions
         ):
             raise ValueError("Specified version {} is not installed on your system".format(version[0:6]))
-        self.dll_path = os.path.join(aedt_versions.installed_versions[version], "nuhertz/FilterSolutionsAPI.dll")
+        self.dll_path = os.path.join(aedt_versions.installed_versions[version], "nuhertz", "FilterSolutionsAPI.dll")
```

Comment:
Good catch !

Reply:
The versions less than 2025.1 do not include the API dll. The dfault values are 200MHz, and 5 GHz. user can set them either using "graph_setup.minimum_frequency" and "graph_setup.maximum_frequency" or directly entering in the "frequency_response" function.